var/spack/repos/builtin/packages/py-neurom/package.py

```diff
@@ -10,8 +10,12 @@ class PyNeurom(PythonPackage):
     """Python library neuron morphology analysis"""
 
     homepage = "https://github.com/BlueBrain/NeuroM"
+    git = "https://github.com/BlueBrain/NeuroM.git"
     url = "https://pypi.io/packages/source/n/neurom/neurom-1.4.10.tar.gz"
 
+    version('develop', branch='master')
+    version('mut_morphio', branch='mut_morphio')
```

Comment:
I'm a little uneasy here, since this is a moving target. I hope there's a plan to merge this at some point?

Reply:
I'm uneasy as well. Yes there is a plan to merge this.